### DISCUSSION TITLE
osu!lazer support sensitivity

### DISCUSSION THREAD

**Discussion by StartZX1** [ppy/osu#17483] [Q&A]
Hi i want soo much play osu lazer, but i can't play. I can't set the sensibility than i in osu!, my sensibility in osu lazer is terrible, i can't put the same config and sensibility of wacom tablet, so when i play, the sensibility is too high, or too low, i use my graphic tablet as mouse( global), and tablet(osu!), someone can help me?

**Comment by frenzibyte** [comment]
I suppose you're referring to sensitivity here, or rather, tablet sensitivity..? that doesn't make sense since tablets work in absolute coordinates.

If you need the same configuration as your external "global" tablet driver, you can disable the in-game tablet driver and use the external one instead (via disabling "Tablet" in game settings).

**Reply by TheFantasticLoki** [reply]
The only thing I can say about the built in driver is that it's hard to match the config from normal otd. The aspect ratio should have 3 numbers after the decimal to match otd cause in lazer 64mm X 36mm goes to 1.79 aspect ratio instead of 1.778 which is the exact ratio of a standard 1920x1080p monitor. This means your area doesn't quite match exactly ratio wise like it does in OTD.
The lock aspect ratio should lock to monitor aspect ratio in the first place instead of the currently set ratio.

But he could also be talking about how the gameplay area feels bigger in lazer than it does in stable making it feel like your sensitivity is wrong even when using the external driver on the exact same settings. But idk it's really hard to understand what he is saying.

**Reply by bdach** [reply]
@TheFantasticLoki please let the user explain themselves rather than guess. your comment is little more than noise.

**Comment by StartZX1** [comment]
how i config this in open tablet driver?

![image](https://user-images.githubusercontent.com/79193364/161384109-3e1b14fe-1c8e-4602-9f01-ec19ff2b136c.png)

for only osu lazer?
also, this is normal?
![image](https://user-images.githubusercontent.com/79193364/161384114-9c878336-12e3-483c-a5d7-66530e899dcd.png)
![image](https://user-images.githubusercontent.com/79193364/161384097-bfe3641a-19f3-4d5d-849f-d42be3e2213a.png)

i can't set the correct settings in osu lazer
the sensibility is horrible
i use my graphic tablet as mouse( in background), tablet ( in osu, and sai)

**Reply by peppy** [reply]
If you are using external tablet "drivers" then you will want to disable tablet support in the game. Or configure it in-game.

**Comment by StartZX1** [comment]
![image](https://user-images.githubusercontent.com/79193364/161385065-98d50dbe-8c34-42a1-a763-f620c3c912d0.png)
1.65x

**Reply by peppy** [reply]
That is not lazer.

**Reply by StartZX1** [reply]
yeah,i just showing my settings

**Reply by bdach** [reply]
you're missing the point, this is not the place to report issues with the stable version of osu!.

**Reply by StartZX1** [reply]
I'm just saying, that I'm not able to activate my osu settings! just like osu laser

**Reply by frenzibyte** [reply]
Simply put, "Sensitivity" is not and won't be a thing in osu!(lazer), so you will need to shorten your area when using it on osu!(lazer) to roughly match how it behaves in osu!(stable) with that 1.65x "sensitivity" you're applying.

**Comment by StartZX1** [comment]
![image](https://user-images.githubusercontent.com/79193364/161385282-5a853f9e-9b51-4f57-8f9a-19eab578242c.png)
could this be?

**Reply by frenzibyte** [reply]
This is irrelevant, see https://github.com/OpenTabletDriver/OpenTabletDriver/issues/1170.

**Comment by StartZX1** [comment]
![image](https://user-images.githubusercontent.com/79193364/161385411-cd55eec2-28c0-4f6b-b2c1-c1fd47da7381.png)
i don't know how i put my wacom settings here

**Reply by frenzibyte** [reply]
Either you disable this and use your external Wacom driver configuration, or enable this and port your Wacom configuration to it (as already mentioned in https://github.com/ppy/osu/discussions/17483#discussioncomment-2491148). 

The metrics system will likely be different between drivers so good luck figuring that out if you intend to port your Wacom configuration to osu!(lazer).

**Reply by StartZX1** [reply]
but I put 1.65 in osu stable and I can play normally, but 1.65 in leisure is slow

**Reply by peppy** [reply]
If you are using external "drivers" or drivers, Make sure you turn off the in-game support and also turn on raw input. If it still doesn't match, then please consider adjusting it until it feels good.
